infra/presubmit.py

```diff
@@ -326,20 +326,21 @@ def run_tests():
   changed_dirs = set()
   for file in get_changed_files():
     changed_dirs.add(os.path.dirname(file))
+
   suite_list = []
   for change_dir in changed_dirs:
     suite_list.append(unittest.TestLoader().discover(change_dir,
                                                      pattern='*_test.py'))
   full_suite = unittest.TestSuite(suite_list)
-  unittest.TextTestRunner().run(full_suite)
+  return not unittest.TextTestRunner().run(full_suite).failures
```

Comment:
please consider the following for better readability:

```
result = unittest.TextTestRunner().run(full_suite)
return not result.failures
```

Reply:
Added.